TCAT/Controllers/HomeViewController.swift

```diff
@@ -134,54 +132,70 @@ class HomeViewController: UIViewController, UITableViewDelegate, UITableViewData
             searchBar.becomeFirstResponder()
         }
     }
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return isBannerShown ? .lightContent : .default
+    }
 
     @objc func reachabilityChanged(note: Notification) {
         let reachability = note.object as! Reachability
         switch reachability.connection {
             case .none:
-                banner.show(queuePosition: .front, on: self)
-                isBannerShown = true
-                UIApplication.shared.statusBarStyle = .lightContent
+                if !isBannerShown {
+                    banner = StatusBarNotificationBanner(title: "No internet connection", style: .danger)
```

Comment:
should we try and put this banner string in `Constants` and then use the same string for my banner?